### PR TITLE
Update changelog, release v0.6.1

### PR DESCRIPTION
**Pull request by Esysc**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -18,6 +20,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - tests: new Task targets `test:unit` (replaces target `test`) and `test:integration` (replaces target `test-integration`).
 - build: cleaner Taskfile.
 
+### Fixed
+
+- return error text code in case of http code 500.
```
Comment (odormond) [inline CHANGELOG.md:25]: The other entries start with an Uppercase letter.
```suggestion
- Return error text code in case of http code 500. 
```

Comment (Esysc) [reply CHANGELOG.md]: Thanks, corrected.

**Review by odormond — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -18,6 +20,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - tests: new Task targets `test:unit` (replaces target `test`) and `test:integration` (replaces target `test-integration`).
 - build: cleaner Taskfile.
 
+### Fixed
+
+- Return error text code in case of http code 500.
```
Comment (marco-m-pix4d) [inline CHANGELOG.md:25]: A "code" is a number, so we don't really return it in the text :-)
Maybe something like "Return standard error description in addition to the error code"

**Review by marco-m-pix4d — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (Esysc) [reply CHANGELOG.md:25]: Thanks for that, I'm doing the change.

**Comment by Esysc** [comment]
Force push to squash commits.

**Pull request merged**